scilpy/tractanalysis/distance_to_centroid.py

```diff
@@ -1,38 +1,536 @@
 # -*- coding: utf-8 -*-
 
+import heapq
+import logging
+import time
+
+from dipy.tracking.metrics import length
+from nibabel.streamlines.array_sequence import ArraySequence
 import numpy as np
+
+from sklearn.preprocessing import MinMaxScaler
+import scipy.ndimage as ndi
 from scipy.spatial import KDTree
+from scipy.spatial.distance import pdist, squareform
+from sklearn.svm import SVC
+
+from scilpy.tractanalysis.streamlines_metrics import compute_tract_counts_map
+from scilpy.tractograms.streamline_operations import \
+    resample_streamlines_num_points, resample_streamlines_step_size
 
 
 def min_dist_to_centroid(bundle_pts, centroid_pts, nb_pts):
```

Comment:
Does this function have the right name ? It seems to return labels, should it be called something that is related to labels ?